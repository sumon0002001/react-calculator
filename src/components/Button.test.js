import React from 'react';
import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import Button from './Button';
import renderer from 'react-test-renderer';

it('Renders the Button page', () => {
    const button = renderer
      .create(<Button />)
      .toJSON();
    expect(button).toMatchSnapshot();
  });