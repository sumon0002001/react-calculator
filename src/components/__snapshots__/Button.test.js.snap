// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the Button page 1`] = `
<button
  className="button"
  type="button"
>
  
</button>
`;

exports[`Renders the Home page 1`] = `
<button
  className="button"
  type="button"
>
  
</button>
`;

exports[`return content from Button matches snapshot 1`] = `
<div>
  <button
    class="button"
    type="button"
  />
</div>
`;
